ArcGISRuntimeSDKQt_CppSamples/Layers/OSM_Layer/OSM_Layer.cpp

```diff
@@ -46,8 +46,13 @@ void OSM_Layer::componentComplete()
   // find QML MapView component
   m_mapView = findChild<MapQuickView*>("mapView");
 
-  // Create a map using the OpenStreetMap basemap
-  m_map = new Map(BasemapStyle::OsmStandard, this);
+  // Create a map
+  m_map = new Map();
+
+  // Create a new OpenStreetMapLayer and add it to the list of operational layers
+  OpenStreetMapLayer* osm = new OpenStreetMapLayer();
+  m_map->operationalLayers()->append(osm);
```

Comment:
Because OpenStreetMapLayer does not support identify, query, selection, or time, I would suggest we add it to the base layers instead. This way it'll be a little more clear as to the intended purpose of this class.

We should also set `m_map`'s and `osm`'s parent to `this` so it will be available through the runtime of the sample. Generally we set a class's parent to `this` any time `new` is used.
```suggestion
  m_map = new Map(this);

  // Create a new OpenStreetMapLayer and add it to the list of basemap base layers
  OpenStreetMapLayer* osm = new OpenStreetMapLayer(this);
  m_map->basemap()->baseLayers()->append(osm);
```